src/components/Step.vue

```diff
@@ -257,6 +257,7 @@ export default class Step extends Vue {
   white-space: nowrap;
   text-overflow: ellipsis;
   overflow: hidden;
+  padding-right: 5px; //avoid label to be truncate due to emphasis style
```

Comment:
Dark magic ahaha

Reply:
Is this supposed to be the size of the ellipsis character? In that case, putting `1em` (size of 1 character) would be a it better because it would adapt to the font-size.

Reply:
i do it!